aim/storage/run_metadata/models.py

```diff
@@ -0,0 +1,82 @@
+from sqlalchemy import Column, Table, ForeignKey
+from sqlalchemy import Integer, Text, Boolean, DateTime
+from sqlalchemy.orm import relationship, backref
+from sqlalchemy.orm import validates
+
+import uuid
+import datetime
+
+from aim.storage.run_metadata.db import Base
+
+
+def get_uuid():
+    return str(uuid.uuid4())
+
+
+def default_to_run_hash(context):
+    return f'Run: {context.get_current_parameters()["run_hash"]}'
+
+
+run_tags = Table(
+    'run_tag', Base.metadata,
+    Column('run_id', Integer, ForeignKey('run.id'), primary_key=True, nullable=False),
+    Column('tag_id', Integer, ForeignKey('tag.id'), primary_key=True, nullable=False)
+)
+
+
+class Run(Base):
+    __tablename__ = 'run'
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    uuid = Column(Text, index=True, unique=True, default=get_uuid)
```

Comment:
I think we should keep either `uuid` or `hash`. Thoughts?

Reply:
the `hash` is effectively used for mapping the sqlite Run entity to the rocksdb data. the `uuid` is somewhat intended for the User (hence URLs constructed based on uuid). It could also be the `run.id`, but in general it's better not to expose  database's internal IDs.  So in this context, you can think of `uuid` as database id. I think we have to keep both.

Reply:
Got it. Taking into account that not all the runs would be stored in sqlite, but the ones which metadata was updated(name, tag, experiment, etc), it probably makes more sense to expose `run.hash` to users. The one that always exists and is stored in rocksdb.